### ISSUE TITLE
PicklingError: Could not pickle the task to send it to the workers.

### ISSUE THREAD

**Issue by entangleddoggie** [tvdboom/ATOM#10]
Hi, great library! It really helped me to accelerate some tasks. Keep it up!

I have encountered this error whenever I put n_jobs different than 1. I am aware this is a common error coming from pickle, but the problem is I am not finding a way to have more information from this error in order to fix it.

I have run this, kept it really simple:
```
import pandas as pd
import atom as atomml

X_train = pd.read_csv('X_train.csv', header=0)
X_test = pd.read_csv('X_test.csv', header=0)
y_train = pd.read_csv('y_train.csv', header=0, squeeze=True)
y_test = pd.read_csv('y_test.csv', header=0, squeeze=True)

atom = atomml.ATOMClassifier(X_train, X_test, y_train, y_test)

atom.run(n_calls=25, n_initial_points=10, verbose=2, n_jobs=2, warnings=False, logger="test2.log", experiment=None, random_state=1)
```

And this is the log:

```
DirectClassifier.run()

Training ===================================== >>
Models: Dummy, GP, GNB, MNB, BNB, CatNB, CNB, Ridge, LR, LDA, QDA, KNN, RNN, Tree, Bag, ET, RF, AdaB, GBM, XGB, LGB, CatB, lSVM, kSVM, PA, SGD, MLP
Metric: f1_weighted


Running BO for Dummy Classification...
Initial point 1 ---------------------------------
Parameters --> {'strategy': 'stratified'}

Exception encountered while running the Dummy model. Removing model from pipeline. 
PicklingError: Could not pickle the task to send it to the workers.

...
```
The log continues and skips almost every model and only manages to finish Gaussian Process and Gaussian Naive Bayes

**Comment by tvdboom** [comment]
Hi. When I run the same example using my own test data it works fine. Is it possible to share the data you are using so I can reproduce the error? The fact that only GP and GNB work suggests that the problem lies with the hyperparameter tuning, since those two models don't run any. Try to run the same code setting `n_calls=0` to check if that works.

It's true that it's hard to debug since the logger does not save the full error traceback for every failed model.   Good feedback point. I'll include it in the next release.

**Comment by tvdboom** [comment]
This error should haven been fixed in v4.9.1. Closing the issue. Feel free to reopen if the issue persist.
